data/core/units/elves/Lord.cfg

```diff
@@ -15,7 +15,9 @@
     advances_to=Elvish High Lord
     cost=36
     usage=fighter
-    description= _ "The nobility of the elves are possessed of merit to match their standing in society. Elvish Lords are the wisest and strongest of their people, and are fearsome in their command of magic."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MAGICAL}+{SPECIAL_NOTES_ARCANE}
+    description= _ "Succession among the Elvish nobility is a highly structured process determined by a number of stringent constraints. The future leaders of the elves have the difficult task of governing the three-fold aspects of Elvish society — warriors and rangers who defend the forests, peaceable civilians who shy away from combat altogether, and the collective group of healers and mystics who study the art of faerie magic. Being able to reconcile the differences between these cliques and effectively lead their people requires considerable personal merit and a deep understanding of all elves, be they warrior, mystic, or pacifist.
+
+Many elves begin training in archery and swordsmanship from a young age and in adolescence, especially promising boys of the appropriate lineage are selected to study under the Enchantresses who have mastered the mystic arts. Since the nature of faerie magic lends itself more to female elves than to male elves, only the most talented learn to temper the volatile way of the faerie and connect themselves to the world of the arcane. Those who prove their personal mettle with sword and sorcery must then learn to apply their newfound powers wisely, for violent conflict is never something to be taken lightly, and just because a battle can be won does not mean it should be fought in the first place. Reconciling potent combat prowess with diplomatic restraint is difficult for most, but those who can do so are granted the title of Lord and become the next generation of Elvish leadership."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MAGICAL}+{SPECIAL_NOTES_ARCANE}
```

Comment:
s/cliques/factions/ ??? Not sure I like cliques here.

Second paragraph, check comma use. "and, in adolescence, ..." ???